llmfoundry/data/finetuning/dataloader.py

```diff
@@ -13,7 +12,9 @@
 from transformers import PreTrainedTokenizerBase
 
 from llmfoundry.data.finetuning.collator import Seq2SeqFinetuningCollator
-from llmfoundry.data.finetuning.tasks import dataset_constructor
+from llmfoundry.data.finetuning.tasks import (_DOWNLOADED_FT_DATASETS_DIRPATH,
```

Comment:
these probably aren't private if we have to import them here?

Reply:
"private" is all fake in python anyway. I'm not too opinionated on which version of this you choose.